cosign/Dockerfile

```diff
@@ -0,0 +1,19 @@
+#Build stage
+FROM golang:1.19 AS build-env
+WORKDIR /cosign
+COPY . .
+RUN make cosign
+
+#Install Cosign 
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+COPY --from=build-env /cosign/cosign /usr/local/bin/cosign
+RUN chown root:0 /usr/local/bin/cosign && chmod g+wx /usr/local/bin/cosign
+
+#Configure home directory
+ENV HOME=/home
+RUN chgrp -R 0 /${HOME} && chmod -R g=u /${HOME}
+
+WORKDIR ${HOME}
+
+# Makes sure's the pod in openshift stays running
+CMD ["tail", "-f", "/dev/null"]
```

Comment:
What is the point of this? Does the pod _need_ to be running constantly?

Reply:
So, I believe that the pod/container (my comment may make it confusing) needs to stay running because of cosign initialize, when using ENTRYPOINT ["cosign"] and when using the initialize cmd, the image will run, initialize, then exit loosing the initialisation, and wont be able to sign anything, the image can still be run they way you described like so

`podman run --name cosign -d quay.io/securesign/cosign:v2.1.1`
`podman exec -it cosign cosign initialize`
`podman exec -it cosign cosign sign.....`
`podman exec -it cosign cosign verfiy.....`
`podman kill cosign` (to stop the running container)
`podman rm cosign` (to delete it)

Thats just what i can see if i am missing something please let me know :)